virtio-devices/src/vhost_user/user_vhost_device.rs

```diff
@@ -0,0 +1,433 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::io::Write as _;
```

Comment:
I don't think this `as _` is idiomatic.  We don't do it anywhere else that I can see.